nlp/text_comparison.py

```diff
@@ -1,5 +1,46 @@
 from sklearn.feature_extraction.text import TfidfVectorizer
 from sklearn.metrics.pairwise import cosine_similarity
+from nlp.greedy_string_tiling import gst
+
+
+def gst_compare_two_texts(a, b, length):
+    first_to_second = gst(a, b, length)
+    second_to_first = gst(b, a, length)
+    similar_chars = 0
+
+    print(first_to_second)
+    for similarity in first_to_second:
+        similar_chars += len(similarity)
+
+    similar_chars = 0
+    print(second_to_first)
+    for similarity in second_to_first:
+        similar_chars += len(similarity)
+
+    return similar_chars / len(a), similar_chars / len(b)
+
+
+def gst_compare_to_many_texts(a, files, length):
```

Comment:
Have you implemented and tested this? I don't see any calls referencing to these functions.